### ISSUE TITLE
unsupported TypeNode TypeOperator

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#142]
upgrading from 
@types/react": "^15.0.20",
to
"@types/react": "^16.0.27",

shows this warning:
```
writeFile [node_modules/@types/react/index.d.ts] test-compile/React.fs
unsupported TypeNode TypeOperator: {pos: 4802, end: 4818, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object]}
unsupported TypeNode TypeOperator: {pos: 4966, end: 4981, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object]}
unsupported TypeNode TypeOperator: {pos: 7157, end: 7173, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object], _children: [object Object],[object Object]}
unsupported TypeNode TypeOperator: {pos: 7233, end: 7248, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object], _children: [object Object],[object Object]}
unsupported TypeNode TypeOperator: {pos: 8313, end: 8329, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object], _children: [object Object],[object Object]}
unsupported TypeNode TypeOperator: {pos: 8529, end: 8544, flags: 0, transformFlags: undefined, parent: [object Object], kind: 170, operator: 127, type: [object Object], _children: [object Object],[object Object]}
```

**Comment by ctaggart** [comment]
This is happening on 0.5.2 and 0.6.0-build.289 with types/react@15.6.14.

```
 yarn list --pattern @types/react --depth 0 -s
└─ @types/react@15.6.14
```
